PolyEngine/Engine/Src/Family.inc

```diff
@@ -0,0 +1,35 @@
+#include "Family.hpp"
+
+namespace Poly
+{
+	template<typename T>
+	size_t Family<T>::identifier()
+	{
+		static size_t value = 0;
+		return value++;
+	}
+
+	template<typename T>
+	template<typename U>
+	size_t Family<T>::type_id()
+	{
+		static const size_t value = identifier();
+		return value;
+	}
+
+	EXPIMP_TEMPLATE template class
+#if (defined(_GAME) && (defined(__GNUC__) || defined(__clang__)))
+
+#else
+	ENGINE_DLLEXPORT
```

Comment:
ENGINE_DLLEXPORT is not defined when compiling for *nix anyway. This ifdef is pointless.

Reply:
Under GCC and clang this macro always turns into: `__attribute__ ((visibility ("default")))`. This attribute causes a compilation error in `_GAME` projects.
IMO this empty `#if` describes what I want to achieve: in `_GAME` under GCC and clang leave it empty. AFAIK this attribute error might be a bug (in GCC) so I want here to highlight that.